packages/vaex-core/vaex/dataframe.py

```diff
@@ -4707,6 +4707,10 @@ def sort(self, by, ascending=True):
         :param str or expression or list of str/expressions by: expression to sort by.
         :param bool or list of bools ascending: ascending (default, True) or descending (False).
         '''
+
+        if len(self) == 0:
+            warnings.warn('Cannot sort empty DataFrame, returning empty DataFrame.')
```

Comment:
```suggestion
```
Do we need to warn here? I think we can just leave it out.

Reply:
Done in new commit (needed to update the test as well, i was checking for the warning.. )

Reply:
I would +1 to there being no warning.